src/openforms/js/components/admin/form_design/registrations/objectsapi/mocks.js

```diff
@@ -0,0 +1,16 @@
+import {rest} from 'msw';
+
+export const BASE_URL = process.env.SB_BASE_URL || '';
+
+export const mockObjecttypesGet = objecttypes =>
+  rest.get(`${BASE_URL}/api/v2/registration/plugins/objects-api/object-types`, (req, res, ctx) => {
+    return res(ctx.json(objecttypes));
+  });
+
+export const mockObjecttypeVersionsGet = (objecttypeUuid, versions) =>
+  rest.get(
+    `${BASE_URL}/api/v2/registration/plugins/objects-api/object-types/${objecttypeUuid}/versions`,
```

Comment:
you can use actual params syntax here as used by express.js (if I understand it correctly)

```suggestion
    `${BASE_URL}/api/v2/registration/plugins/objects-api/object-types/:uuid/versions`,
```
so you then don't need to pass in the object type UUID in your mock setup in the stories.